lib/metasploit/framework/mssql/metasploit_mixin.rb

```diff
@@ -0,0 +1,118 @@
+require 'rex/proto/mssql/client'
+
+module Metasploit
+module Framework
+module MSSQL
+
+    # A base mixin of useful mssql methods for parsing structures etc
+    module MetasploitMixin
```

Comment:
taking name suggestions :)

Reply:
We just swapped this file to be called `MetasploitMixin` temporarily to help make things easier to understand :+1:

I think once you move the mssql class and clientmixin into the rex proto namespace, you can consolidate the concept of `MetasploitMixin` into the `lib/msf/core/exploit/remote/mssql.rb` implementation,  and then you should be in a position to revert the modules back to their original files before changes were made

i.e. Transitioning to having a fully independent mssql client should be a backwards compatible change for modules